aptos-move/framework/aptos-framework/sources/account.move

```diff
@@ -183,11 +184,7 @@ module aptos_framework::account {
         *&borrow_global<Account>(addr).authentication_key
     }
 
-    public entry fun rotate_authentication_key(account: &signer, new_auth_key: vector<u8>) acquires Account {
```

Comment:
Why delete this now?

Reply:
we had planned to... it is extremely unsafe... we are using signed authorization in move to validate rotation.

Reply:
Okay